pkgs/tools/misc/blahaj/default.nix

```diff
@@ -1,6 +1,9 @@
 { lib
+, stdenv
 , crystal
 , fetchFromGitHub
+, mt ? true
+, static ? stdenv.targetPlatform.isStatic
```

Comment:
Is it needed to add this to all crystal packages? I think it should be done in crystal builder.

Reply:
as this is done in the project specific makefile, it would have to be done in some other way at least.